Casks/suspicious-package.rb

```diff
@@ -10,8 +10,8 @@
   livecheck do
     url "https://www.mothersruin.com/software/SuspiciousPackage/data/SuspiciousPackageVersionInfo.plist"
     strategy :page_match do |page|
-      v = page.match(/<string>(\d+(?:\.\d+)*)/i)
-      id = page.match(/<string>(\d{2,})/i)
+      v = page.match(/CFBundleShortVersionString.*?(\d+(?:\.\d+)*)/i)
+      id = page.match(/CFBundleVersion.*?(\d+)/i)
```

Comment:
By default "." does not match "\n" character, so you can enable dot match newline by using "/m" modifier or you can manually match "\n".

```suggestion
      v = page.match(/CFBundleShortVersionString.*?(\d+(?:\.\d+)*)/im)
      id = page.match(/CFBundleVersion.*?(\d+)/im)
```

```suggestion
      v = page.match(/CFBundleShortVersionString.*?\n.*?(\d+(?:\.\d+)*)/i)
      id = page.match(/CFBundleVersion.*?\n.*?(\d+)/i)
```

Both versions do work as below:

```
ykursadkaya@YKKs-MacBook-Air: ~% brew livecheck --cask suspicious-package
suspicious-package : 3.6,736 ==> 3.6,736
```

Reply:
Not the first case of this. It might be worth adding a specific strategy for plists, as they’re *the* way to define app versions.

Reply:
Thank you @ykursadkaya -- I was sure I tried the `\n` switch!